### DISCUSSION TITLE
Porting LPC55S69

### DISCUSSION THREAD

**Discussion by blairs123** [micropython/micropython#12341] [Core Development]
Hi there,

I have started porting MicroPython to the LPC55S69. Steps so far include:
- Starting with the "minimal" build from the repo
- Adding the SDK for the uC to the project and modifying the makefile
- Wrote some generic code to light an LED on the LPCXpresso55S69 dev board
- Simple REPL working tested with "print("hello world")

Where im currently struggling:
- When running the REPL over UART to my computer I cant seem to import any basic libraries e.g. math
- I have defined MICROPY_PY_MATH in the mpconfigport.h
- I have run the steps outlined, e.g. make -C mpy-cross, then make my project
- I receive a bin file that lights the LED, runs the basic REPL but cannot import math

I'm new to MicroPython and porting altogether, so I may have missed some simple steps.

Any help greatly appreciated.
Blair

**Comment by blairs123** [comment]
UPDATE: I forgot to also to define MICROPY_FLOAT_IMPL, once defined it worked, also had to add a few libm files

**Comment by MaureenHelm** [comment]
You can also try the Zephyr port. I don't have this board anymore to verify, but the MicroPython Zephyr port used to work on it.
